portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,23 @@ function addRandomFacts() {
   factContainer.innerText = fact;
 }
 
-/** Fetches the content in DataServlet and display on Home page */
+/** Fetches comments from the servers and adds them to the DOM. */
 function getData() {
-  fetch('/data').then(response => response.text()).then((data) => 
-  {document.getElementById("data-container").innerText = data;});
+  fetch('/data?numComment='+document.getElementById("numComment").value).then(response
+   => response.json()).then((data) => {
```

Comment:
I would suggest use 4 spaces for breaking a line. And I would suggest a new line from ```.then```
```
fetch('/data?numComment='+document.getElementById("numComment").value).
    then(response=> response.json()).then((data) => {
  ...
}
```